### PR TITLE
Updm rule

### PR DESCRIPTION
**Pull request by MikeMillerGIS**

### CODE REVIEW

File: HelperTools/Apply_All_Industry_Rules.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        details = details.strip()
+        if param not in comments_to_parameter:
+            print(f"{param} not defined in lookup")
+            continue
+        if param == 'Assigned To':
+            if is_un:
+                kwargs[comments_to_parameter[param]] = os.path.join(workspace, details)
+                fcs.add(os.path.join(workspace, details))
+            else:
+                kwargs[comments_to_parameter[param]] = details
```
Comment (ted-howard) [inline HelperTools/Apply_All_Industry_Rules.py:136]: lets check if class exists

File: HelperTools/Apply_All_Industry_Rules.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    kwargs['script_expression'] = script_expression
+    if 'NextSequenceValue' in script_expression:
+        seq_names = pat.findall(script_expression) or []
+        if not seq_names:
+            print('***** Could not parse sequences, make sure to the dict format for definition')
+        else:
+            all_seq.extend([dict(seq_name=seq_name, seq_start_id=1, seq_inc_value=1) for seq_name in seq_names])
+
+    if 'type' not in kwargs:
+        print(f'***** Type is missing from {path}')
```
Comment (ted-howard) [inline HelperTools/Apply_All_Industry_Rules.py:162]: would be good to check if all required keys you're adding in the while loop (in_table, name, type, triggering_events if calculation, and several others depending on rule type) are in kwargs before proceeding

File: HelperTools/Apply_All_Industry_Rules.py (hunk 1 above)
Comment (MikeMillerGIS) [reply HelperTools/Apply_All_Industry_Rules.py]: updated

File: HelperTools/Apply_All_Industry_Rules.py (hunk 2 above)
Comment (MikeMillerGIS) [reply HelperTools/Apply_All_Industry_Rules.py]: updated

**Pull request merged**